app/src/main/res/xml/developer_preferences.xml

```diff
@@ -349,6 +349,11 @@
 
     <PreferenceCategory android:title="A/B tests">
 
+        <org.wikipedia.settings.IntPreference
+            style="@style/IntPreference"
+            android:key="ab_test_notificationIcons"
+            android:title="Test group for in-app Notification icons (0, 1, or 2)" />
```

Comment:
I am seeing my test group id is `-1519020059`, which is a negative number. Is this expected?

Reply:
Yes, that is fine. The test group is your group id _modulo 3_.